subgraphs/reviews/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM python:3.10
+
+# install PDM
+RUN pip install -U pip setuptools wheel
+RUN pip install pdm
+
+# copy files
+COPY pyproject.toml pdm.lock /project/
+COPY . /project
+
+WORKDIR /project
+RUN pdm install --prod --no-lock --no-editable
+
```

for faster docker down of the container image
- see https://github.com/apollographql/router/issues/320#issuecomment-1209696958 for reference

```suggestion
STOPSIGNAL SIGINT
```

---

ah, neat! I'll test this and push it 😊